src/main/webapp/js/feedbackResponseComments.es6

```diff
@@ -347,25 +347,28 @@ function registerResponseCommentsEvent() {
 
     const clickHandlerMap = new Map();
     clickHandlerMap.set('.show-frc-add-form',
-            [showResponseCommentAddForm, ['recipientindex', 'giverindex', 'qnindex', 'sectionindex']]);
+        [showResponseCommentAddForm, ['recipientindex', 'giverindex', 'qnindex', 'sectionindex']]);
```

Comment:
Tried making some changes locally on my end, and it looks like

```javascript
    clickHandlerMap.set(
            '.show-frc-add-form', [showResponseCommentAddForm,
                    ['recipientindex', 'giverindex', 'qnindex', 'sectionindex']]);
```

seems to work. Not sure why it's throwing out errors in the other case but doing it this way should be fine.